TestStack.Dossier/Suppliers/DefaultValueSupplier.cs

```diff
@@ -16,5 +18,16 @@ public class DefaultValueSupplier : IAnonymousValueSupplier
         {
             return default(TValue);
         }
+
+        /// <inheritdoc />
+        public object GenerateAnonymousValue(AnonymousValueFixture any, Type type, string propertyName)
+        {
+            // See stackoverflow: http://stackoverflow.com/questions/325426/programmatic-equivalent-of-defaulttype
```

Comment:
Happy with this, but I suspect it would never have been called since the value types would have been picked up before this supplier with the default ordering.